### ISSUE TITLE
Devtools protocol is not working on the latest version. 

### ISSUE THREAD

**Issue by pjcalvo** [webdriverio/webdriverio#5634]
[//]: # NOTE: This repository only maintains packages that are listed in the [Readme](https://github.com/webdriverio/webdriverio/blob/master/README.md#packages). Please make sure that your issue is directly caused by one of these packages and if not file an issue in the correct 3rd party package repository.

**Environment (please complete the following information):**
 - **WebdriverIO version: 6.3.0**
 - **WDIO Testrunner**
 - **[sync/async] both**
 - **Node.js version: [10 && 14]**
 - **NPM version: 6.3.14**
 - **Browser name and version:** [e.g. Chrome 68]
 - **Platform name and version:** [e.g. Windows 10]
 - **Additional wdio packages used (if applicable):@wdio/devtools service**

**Config of WebdriverIO**
```
exports.config = {
    runner: 'local',
    automationProtocol: 'devtools',
    specs: [
        './test/specs/**/*.js'
    ],
    capabilities: [{
        browserName: 'chrome',
    }],
    services: ['devtools'],
...
```

**Describe the bug**

On the last version of webdriverIO which ues `puppeteer-code > 5.0.1` webdriverIO is not able to execute tests using the devtools protocol. 
If I use an older version i.e. webdriver 6.2 which uses `puppeteer-code 4.x.x` then the same execution works fine.

Test execution breaks right after initializing the browser.

**To Reproduce**

1) Create a new folder
2) Install the latest @wdio/cli 
      `npm install --save-dev @wdio/cli`
3) Setup the default config, but choose `devtools service`
     `npx wdio`
4) Make sure that your config has devtools as the automation-protocol, and only uses devtools service. (Look at the config above)
5) Execute the tests
    `npx wdio`

**Expected behavior**
Tests should run just fine.

**Log**
2020-07-17T17:14:11.562Z INFO @wdio/local-runner: Run worker command: run
2020-07-17T17:14:11.567Z DEBUG @wdio/local-runner:utils: init remote session
2020-07-17T17:14:11.569Z INFO webdriverio: Initiate new session using the ./protocol-stub protocol
2020-07-17T17:14:11.676Z DEBUG @wdio/utils:initialiseServices: initialise service "devtools" as NPM package
2020-07-17T17:14:11.984Z DEBUG @wdio/local-runner:utils: init remote session
2020-07-17T17:14:11.985Z INFO webdriverio: Initiate new session using the devtools protocol
2020-07-17T17:14:11.985Z INFO devtools: Launch Google Chrome with flags: --disable-extensions --disable-background-networking --disable-background-timer-throttling --disable-backgrounding-occluded-windows --disable-sync --metrics-recording-only --disable-default-apps --mute-audio --no-first-run --disable-hang-monitor --disable-prompt-on-repost --disable-client-side-phishing-detection --password-store=basic --use-mock-keychain --disable-component-extensions-with-background-pages --disable-breakpad --disable-dev-shm-usage --disable-ipc-flooding-protection --disable-renderer-backgrounding --enable-features=NetworkService,NetworkServiceInProcess --disable-features=site-per-process,TranslateUI,BlinkGenPropertyTrees --window-position=0,0 --window-size=1200,900
2020-07-17T17:14:13.031Z INFO devtools: Connect Puppeteer with browser on port 58391
2020-07-17T17:14:13.835Z ERROR @wdio/runner: Error: Couldn't initialise "/Users/pjcalvo/Documents/wdioissue/node_modules/devtools/build/commands/performActions".
Error: Cannot find module 'puppeteer-core/lib/cjs/common/USKeyboardLayout'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:636:15)
    at Function.Module._load (internal/modules/cjs/loader.js:562:25)
    at Module.require (internal/modules/cjs/loader.js:692:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at Object.<anonymous> (/Users/pjcalvo/Documents/wdioissue/node_modules/devtools/build/commands/performActions.js:8:25)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
    at safeRequire (/Users/pjcalvo/Documents/wdioissue/node_modules/@wdio/utils/build/utils.js:147:11)
    at new DevToolsDriver (/Users/pjcalvo/Documents/wdioissue/node_modules/devtools/build/devtoolsdriver.js:44:59)
    at Function.newSession (/Users/pjcalvo/Documents/wdioissue/node_modules/devtools/build/index.js:57:20)
    at process._tickCallback (internal/process/next_tick.js:68:7)

**Additional context**

> Please use the correct markdown

**Comment by christian-bromann** [comment]
Thanks for reporting. This happened due to a change in the Puppeteer file structure. I will make a follow up PR to the Puppeteer project to ensure that the keyboard can be imported safely moving on.

**Comment by pjcalvo** [comment]
So this is already fixed in the 6.3.3?

**Comment by christian-bromann** [comment]
> So this is already fixed in the 6.3.3?

Yes

**Comment by christian-bromann** [comment]
Does it work on your end now?
